conbench/templates/benchmark-entity.html

```diff
@@ -104,6 +104,17 @@
               {% endif %}
             </li>
           {% endfor %}
+          {% if benchmark.optional_info %}
+            <li class="list-group-item active">Additional information</li>
+            {% for k,v in benchmark.optional_info.items() %}
+              <li class="list-group-item" style="overflow-y: auto;">
+                <b>{{ k }}</b>
+                  {% if v is not none %}
+                    <div align="right" style="display:inline-block; float: right;">{{ v }}</div>
+                  {% endif %}
+              </li>
+            {% endfor %}
+          {% endif %}
```

Comment:
Ddid you try to look at this in UI locally?

Reply:
Not yet, but I need to

Reply:
Ok, I did and it looks fine! Thank you again for explicitly pointing me to the population script — I could not find that yesterday evening when I was struggling to put data there.